scripts/ci/libraries/_md5sum.sh

```diff
@@ -152,22 +152,3 @@ function md5sum::check_if_docker_build_is_needed() {
         fi
     fi
 }
-
```

Comment:
We do not need to pull the image at all now - buildkit caching is checked without pulling the image first.